### DISCUSSION TITLE
Implement a Github bot to make code review easier

### DISCUSSION THREAD

**Discussion by one2code** [EOT-Event-Organizer-s-Toolbox/eot-codebase#92] [Ideas]
I was thinking we should implement a Github bot that will automatically take a submitted branch and paste it as a comment. For example:

`git checkout -b *user submitted branch*`

`git pull https://github.com/*user that submitted the pr*/eot-codebase.git *branch name *`

This will make the project more accessible to contributors over the long-term, and help avoid short-term pain. Thoughts @0916dhkim @DaleOrders? I have a base template to get started on this Saturday. @DaleOrders if you are interested in pair programming this weekend, let me know. I've reviewed some Github Actions in my code brigade to get my feet wet and gain some familiarity with how they're written, and how they work. Would love to get this going for Saturday.

**Comment by 0916dhkim** [comment]
GitHub cli supports this feature, and I use it all the time at work!
```
gh pr checkout [number]
```
https://cli.github.com/
